biomage/account/account.py

```diff
@@ -267,6 +277,92 @@ def create_users_list(user_list, header, input_env, aws_profile, overwrite):
             out.write("%s,%s,%s\n" % (full_name, email, password))
 
 
+@click.command()
+@click.option(
+    "--user_list",
+    required=True,
+    help="Path to the user list csv file containing user and email for the new accounts.",
+)
+@click.option(
+    "--experiment_name",
+    required=True,
+    help="Experiment name to be created",
+)
+@click.option(
+    "--samples_path",
+    required=True,
+    help="Local path to the samples for upload",
+)
+@click.option(
+    "--instance_url",
+    required=False,
+    default="https://api-default.scp-staging.biomage.net/",
+    help="URL of the cellenics api",
+)
+@click.option(
+    "-p",
+    "--aws_profile",
+    required=False,
+    default=DEFAULT_AWS_PROFILE,
+    show_default=True,
+    help="The name of the profile stored in ~/.aws/credentials to use.",
+)
+@click.option(
+    "--admin_email",
+    required=True,
+    help="admin email for cognito",
+)
+@click.option(
+    "--admin_password",
+    required=True,
+    help="admin password for cognito",
+)
+def create_process_experiment_list(
+    experiment_name,
+    user_list,
+    samples_path,
+    instance_url,
+    aws_profile,
+    admin_email,
+    admin_password,
+):
+    """
+    Creates users, using the user_list file.
+    Creates experiment, uploads samples and processes the projet
+    for each row in the user_list file.
+    The file should be in csv format.
+    The first column should be the full_name in the format: first_name last_name
+    The second column should be the email.
+    E.g.: Arthur Dent, arthur_dent@galaxy.gl
+    """
+
+    cognito_pool = COGNITO_STAGING_POOL
+
+    # creating the users
+    print('Creating users from the csv file')
+    _create_users_list(user_list, None, "staging", aws_profile, False)
+
+    session = boto3.Session(profile_name=aws_profile)
+    client = session.client("cognito-idp")
+    created_users = pd.read_csv(user_list + ".out", header=None, quoting=csv.QUOTE_ALL)
+
+    # creating the experiment and uploading samples
+    admin_connection = bpi.Connection(admin_email, admin_password, instance_url)
+
+    print('Creating and uploading samples for the experiment as admin')
+    experiment = admin_connection.create_experiment(experiment_name)
+
+    experiment.upload_samples(samples_path)
+    print('Cloning and running the experiment for each user')
```

Comment:
It's great htat you added more comments, but I think it is still important to also print the experiment ids created (and for which user they were created) so that, as admins, we can easily track that information